### DISCUSSION TITLE
请问能删除播放器的默认快捷键么

### DISCUSSION THREAD

**Discussion by nyano** [the1812/Bilibili-Evolved#2401] [功能建议 / Ideas]
什么w d快捷键好烦。。老是误按
我看到了快捷键扩展修改功能
但是只能覆盖默认快捷键。。不指定到默认快捷键就会变成默认播放器的快捷键。
能不能直接删除或者禁用默认部分快捷键？比如我只想保留左右键快进，空格暂停等。其他全部删掉
谢谢

**Comment by Geeyun-JY3** [comment]
可以的：https://github.com/the1812/Bilibili-Evolved/blob/preview/src/video/keymap/key-bindings.md
https://github.com/the1812/Bilibili-Evolved/discussions/2231#discussioncomment-1275960
